src/Features/Clonable.php

```diff
@@ -135,9 +135,11 @@ private function cloneRelations(CommonDBTM $source, bool $history): void
                 }
                 $origin_id = $relation_item->getID();
                 $itemtype = $relation_item->getType();
-                $cloned[$itemtype][$origin_id] = $relation_item->clone($override_input, $history);
-                $relation_item->getFromDB($cloned[$itemtype][$origin_id]);
-                $relation_newitems[] = $relation_item;
+                if (method_exists($relation_item, 'clone')) {
```

Comment:
Seems a real error if `clone()` method des not exists.

Reply:
I added an else